<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoginDialog</class>
 <widget class="QDialog" name="LoginDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="LoginLayout" rowstretch="0">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="LoginPage">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <layout class="QGridLayout" name="LabelLayout">
         <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="LoginIcon">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string/>
           </property>
           <property name="scaledContents">
            <bool>true</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <spacer name="Labe_horizontalSpace_left">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1">
          <spacer name="Label_verticalSpacer_top">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="Label_verticalSpacer_buttom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Minimum</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="2">
          <spacer name="Label_horizontalSpacer_right">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="LoginText">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>LOGIN/登录</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="2" column="0">
        <layout class="QGridLayout" name="ButtonLayout" rowstretch="0,0" columnstretch="1,1,1,1">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="0" column="3">
          <spacer name="Button_horizontalSpacer_right">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0">
          <spacer name="Button_horizontalSpacer_left">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" colspan="2">
          <layout class="QVBoxLayout" name="ButtonGrpup">
           <property name="spacing">
            <number>20</number>
           </property>
           <item>
            <widget class="QPushButton" name="Login">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>登录</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Register">
             <property name="font">
              <font>
               <family>Microsoft JhengHei UI Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string>注册</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="1" column="1" colspan="2">
          <spacer name="Button_verticalSpacer_buttom">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
       <item row="1" column="0">
        <layout class="QGridLayout" name="EditLayout" rowstretch="2,0,0,0" columnstretch="1,2,1">
         <property name="horizontalSpacing">
          <number>10</number>
         </property>
         <property name="verticalSpacing">
          <number>20</number>
         </property>
         <item row="3" column="1">
          <layout class="QFormLayout" name="HintLayout">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>6</number>
           </property>
           <item row="0" column="0">
            <widget class="QCheckBox" name="Remmber">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="text">
              <string>记住密码</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLabel" name="Forget">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>10</pointsize>
               <underline>true</underline>
              </font>
             </property>
             <property name="text">
              <string>忘记密码</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item row="1" column="0" colspan="2">
            <widget class="QLineEdit" name="HintEdit">
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>10</pointsize>
              </font>
             </property>
             <property name="frame">
              <bool>false</bool>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="2" rowspan="4">
          <spacer name="Edit_horizontalSpacer_right">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" rowspan="4">
          <spacer name="Edit_horizontalSpacer_left">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="1" rowspan="3">
          <layout class="QGridLayout" name="EditGridLayout">
           <property name="horizontalSpacing">
            <number>0</number>
           </property>
           <property name="verticalSpacing">
            <number>20</number>
           </property>
           <item row="0" column="0">
            <widget class="QLineEdit" name="NameEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLineEdit" name="NumEdit">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="font">
              <font>
               <family>Microsoft YaHei UI Light</family>
               <pointsize>14</pointsize>
              </font>
             </property>
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="RegisterPage">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,9">
       <item>
        <layout class="QGridLayout" name="RegisterLayout">
         <item row="1" column="1" alignment="Qt::AlignHCenter|Qt::AlignVCenter">
          <widget class="QLabel" name="RegisterText">
           <property name="font">
            <font>
             <family>Microsoft YaHei UI</family>
             <pointsize>15</pointsize>
            </font>
           </property>
           <property name="text">
            <string>注册账户</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1" alignment="Qt::AlignHCenter|Qt::AlignBottom">
          <widget class="QLabel" name="RegisterIcon">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>100</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
         <item row="0" column="2" rowspan="2">
          <spacer name="Register_Icon_horizontalSpacer_right">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="0">
          <spacer name="Register_Icon_horizontalSpacer_left">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="0" column="0" alignment="Qt::AlignTop">
          <widget class="QToolButton" name="ReturnButton">
           <property name="text">
            <string>重登录</string>
           </property>
           <property name="icon">
            <iconset resource="src.qrc">
             <normaloff>:/Image/return.png</normaloff>:/Image/return.png</iconset>
           </property>
           <property name="iconSize">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
           <property name="toolButtonStyle">
            <enum>Qt::ToolButtonTextBesideIcon</enum>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QFrame" name="InformationLayout">
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <property name="spacing">
           <number>0</number>
          </property>
          <item row="0" column="0">
           <layout class="QGridLayout" name="InformationBox">
            <property name="horizontalSpacing">
             <number>10</number>
            </property>
            <property name="verticalSpacing">
             <number>20</number>
            </property>
            <item row="0" column="0" colspan="2">
             <widget class="QLabel" name="UserName">
              <property name="text">
               <string>用户名：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0" colspan="2">
             <widget class="QLabel" name="PhoneNumber">
              <property name="text">
               <string>手机号：</string>
              </property>
             </widget>
            </item>
            <item row="2" column="2">
             <widget class="QLineEdit" name="PhoneNumberEdit">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>15885675246</string>
              </property>
             </widget>
            </item>
            <item row="2" column="3">
             <layout class="QHBoxLayout" name="CodeLayout">
              <property name="spacing">
               <number>10</number>
              </property>
              <property name="rightMargin">
               <number>6</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="Code">
                <property name="text">
                 <string>验证码：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="CodeEdit">
                <property name="font">
                 <font>
                  <pointsize>14</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QToolButton" name="CodeButton">
                <property name="text">
                 <string>发送验证码</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item row="3" column="0" colspan="2">
             <widget class="QLabel" name="NewPassword">
              <property name="text">
               <string>创建密码：</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0" colspan="2">
             <widget class="QLabel" name="AplayPassword">
              <property name="text">
               <string>确认密码：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="0" colspan="2">
             <widget class="QLabel" name="Email">
              <property name="text">
               <string>邮箱：</string>
              </property>
             </widget>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLineEdit" name="EmailEdit">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Admin@outlook.com</string>
              </property>
             </widget>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLineEdit" name="UserEdit">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>Admin</string>
              </property>
             </widget>
            </item>
            <item row="3" column="2" colspan="2">
             <widget class="QLineEdit" name="NewPasswordEdit">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>123456</string>
              </property>
             </widget>
            </item>
            <item row="4" column="2" colspan="2">
             <widget class="QLineEdit" name="ApplyPasswordEdit">
              <property name="font">
               <font>
                <pointsize>14</pointsize>
               </font>
              </property>
              <property name="text">
               <string>123456</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="1" column="0">
           <layout class="QHBoxLayout" name="Register_Button_horizontalLayout">
            <item>
             <spacer name="Register_Button_horizontalSpacer_left">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="RegisterButton">
              <property name="text">
               <string>注册账号</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="Register_Button_horizontalSpacer_right">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeType">
               <enum>QSizePolicy::Minimum</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="src.qrc"/>
 </resources>
 <connections/>
</ui>
